### PR TITLE
Small fixes balancers section

### PR DESCRIPTION
**Pull request by HonkingGoose**
Changes this PR makes:

- Improve grammar.
- Simplify sentences.
- Improve inherent belt balancing explanation.
- Move a sentence into the relevant section.
- Add punctuation at the end of sentences.

### CODE REVIEW

File: src/app/views/cheat-sheets/balancers/balancers.component.html [hunk 1]
```diff
@@ -33,24 +33,22 @@
...
-                    It's generally recommended to make builds inherently balanced targeting a complete use of a lane.
+                    Make a build that's inherently balanced, by letting the assembly machines consume a full belt of items.
                 </li>
                 <li>
                     Balancers can still be useful regardless; most common are the
                     <app-blueprint-clipboard [url]="sheetData?.commonBalancers[0].raw"></app-blueprint-clipboard>
                     <a href="{{sheetData?.commonBalancers[0].url}}" target="_blank" rel="noopener">4<i class="fas fa-long-arrow-alt-right"></i>4</a> and
                     <app-blueprint-clipboard [url]="sheetData?.commonBalancers[1].raw"></app-blueprint-clipboard>
-                    <a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a> throughput unlimited.
+                    <a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a> throughput unlimited varieties.
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/balancers/balancers.component.html:45]: Maybe remove the " throughput unlimited varieties." since you elaborate after

Comment (HonkingGoose) [reply src/app/views/cheat-sheets/balancers/balancers.component.html:45]: I'm not sure that I understand what you want me to do...

1. Do you want me to remove this line?
>Being throughput unlimited input/output, the amount can vary thus those two cover most use cases.

2. Or do you want me to remove the second link to the 8x8 balancers (_and remove the entry from balancers.json_) `<a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a>`?

3. Or do you want me to remove this text?
> throughput unlimited varieties

**Comment by HonkingGoose** [comment]
Hi @deniszholob,

I found another cool thing that GitHub lets you do!
A maintainer of a fork, like yourself, can make a change in the pull request directly as a "Commit suggestion".

The steps are explained here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

After you've have committed the suggestion(s), I can (batch) include them in the PR, and you'll get credit for your change(s).
This is easier and faster than communicating back and forth on what is the precise change the maintainer wants. 😉

Greetings, HonkingGoose

File: src/app/views/cheat-sheets/balancers/balancers.component.html (hunk 1 above)
Comment (HonkingGoose) [reply src/app/views/cheat-sheets/balancers/balancers.component.html:45]: Also read https://github.com/deniszholob/factorio-cheat-sheet/pull/268#issuecomment-674359617 for tips on how to apply the change from within the Pull Request interface as a maintainer of the project. 😉
I think you'll like it.

**Comment by HonkingGoose** [comment]
I've applied the suggestion you gave me.
I think this is ready for a final review and merge. 😄

File: src/app/views/cheat-sheets/balancers/balancers.component.html [hunk 2]
```diff
@@ -45,12 +45,14 @@
                     <a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a> throughput unlimited.
                 </li>
                 <li>
-                    Being throughput unlimited input/output amount can vary thus those two cover most cases.
+                    Being throughput unlimited, the input/output amount can vary, thus those two cover most cases.
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/balancers/balancers.component.html:48]: Baaah, the suggestion thing created duplicate lines :/

Comment (HonkingGoose) [reply src/app/views/cheat-sheets/balancers/balancers.component.html:48]: I've fixed it.

**Pull request merged**